game.py

```diff
@@ -404,6 +434,14 @@ def check_next_turn(previos_player, round_id):
         return redirect(url_for('app_game.end_game', round_id=round_id))
     return str(player_shift != previos_player or len(players) == len(rounds[round_id - 1].cards))
 
+@app_game.route('/check_end_game')
+@login_required
+def check_end_game():
+    global game_started
+    if game_started == False:
```

Comment:
simplify this, 
can we refactor this for once and all and make sure that this never happens